src/control/server/ctl_system.go

```diff
@@ -253,7 +294,7 @@ func resolveLeaderMemberAddr(leader *IOServerInstance, members system.Members) (
 // with a harness managing one or more data-plane instances (DAOS system members).
 //
 // TODO: specify the ranks managed by the harness that should be started.
-func (svc *ControlService) startMembers(ctx context.Context, leader *IOServerInstance) (map[string]error, error) {
+func (svc *ControlService) startMembers(ctx context.Context, leader *IOServerInstance) (system.HarnessResults, error) {
```

Comment:
From a symmetry perspective, it would be nice to try to unify the returned structure from each of these operations. It seems like the same structure could be used for stop/start/query.